src/gluonts/dataset/pandas.py

```diff
@@ -50,15 +53,15 @@ class PandasDataset:
         Frequency of observations in the time series. Must be a valid pandas
         frequency.
     feat_dynamic_real
-        List of column names that contain dynamic real features.
+        list of column names that contain dynamic real features.
     past_feat_dynamic_real
-        List of column names that contain dynamic real features only available
+        list of column names that contain dynamic real features only available
```

Comment:
not sure I agree, it's a word (not a code expression) starting a sentence

Reply:
I think I accidentally changed this as well.